### PR TITLE
[NEW] 1st draft of account_due_list integration with account_payment_partner

### PR DESCRIPTION
**Pull request by mileo**
Hi All,

I'm following the mail: [Banking-addons-drivers] banking refactoring next step

As Raphael said, here in Brazil "everything is paid in 10x or so".

We are thinking in a way about meet the needs of account_payment_extension.

This is a 1 hour work scratch about integration of account_due_list and account_payment_partner like as account_payment_extension do with payment.type and account.move.line.

I'll be happy to receive any feedback
Thanks in advance

Sorry for the two incorrects pull requests.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please solve little flake8 issues?:

```
./account_payment_move_line/model/account_move_line.py:33:1: W391 blank line at end of file
./account_payment_move_line/model/account_invoice.py:31:80: E501 line too long (81 > 79 characters)
./account_payment_move_line/model/account_invoice.py:35:80: E501 line too long (80 > 79 characters)
./account_payment_move_line/model/account_invoice.py:39:5: E265 block comment should start with '# '
./account_payment_move_line/model/account_invoice.py:39:5: E303 too many blank lines (2)
./account_payment_move_line/model/account_invoice.py:56:22: W292 no newline at end of file
```

File: account_payment_move_line/model/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   #     Can be like that instead of action_move_create
+   #     '''
+   #     for line in move_lines:
+   #         if line[2]['account_id'] == invoice_browse.account_id.id:
+   #             line[2]['payment_mode_id'] = invoice_browse.payment_mode_id.id
+   #     return move_lines
+
+    def action_move_create(self, cr, uid, ids, context=None):
+        '''
+        ICopy from OCA/account_payment/account_payment_extension
```
Comment (rvalyi) [inline account_payment_move_line/model/account_invoice.py]: typo "ICopy"

File: account_payment_move_line/model/__init__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Account Payment Partner module for OpenERP
+#    Copyright (C) 2014 Akretion (http://www.akretion.com)
+#    @author Alexis de Lattre <alexis.delattre@akretion.com>
```
Comment (rvalyi) [inline account_payment_move_line/model/__init__.py]: better you put your own copyright here

File: account_payment_move_line/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    'version': '0.1',
+    'category': 'Banking addons',
+    'license': 'AGPL-3',
+    'summary': 'Adds payment mode on move lines',
+    'description': """
+Account Move Line
+=======================
+
+This module add the field payment mode in account move line
+
```
Comment (rvalyi) [inline account_payment_move_line/__openerp__.py:35]: add the field payment mode -> adds the payment mode field

**Comment by rvalyi** [comment]
Hello @mileo 

My review is still superficial at this stage, but basically you should not bring the account_due_list module into the banking repo in v7 as it is already present in OCA account_payment repo. A migration of this module is planned but only in v8, otherwise you would create a duplicate.

To make the module pass, you should tweak the .travis.yml file to add a dependency (see how other project do it).

That being said, I'm not too sure we should create a cross repo dep at this level for a 20 lines modules in v7. I have no strong opinion on this, but may be just keep the module in a kmee repo in v7 and then in v8 have it inside the banking repo once account_due_list is migrated (you can help).

I'll try to review the concepts of your PR better though, but it seems to make sense

**Comment by pedrobaeza** [comment]
As this one seems to be abandoned, I close. If I'm wrong, please feel free to reopen or comment.

**Comment by mileo** [comment]
LGTM

**Pull request closed without merging**